packages/flex-dev-utils/src/axios.ts

```diff
@@ -1,9 +1,16 @@
 import axios, { AxiosRequestConfig, AxiosInstance } from 'axios';
 import MockAdapter from 'axios-mock-adapter';
 
+// @ts-ignore
```

Comment:
why doesn't ts like this?

Reply:
It doesn't have the type definition for these files.